virtue/run.py

```diff
@@ -147,6 +148,18 @@ def start_container(conf, docker_client, args):
             print("Starting service %s ..." % (container), end='', flush=True)
             container_obj.start()
             print("[OK]")
+            
+            # Copy network rules into the docker container
+            print("Copying network rules...", end='', flush=True)
+            tar = tarfile.open('networkRules.tar',mode='w')
+            try:
+                tar.add('/etc/networkRules',arcname='networkRules')
+            finally:
+                tar.close()
```

Comment:
Should this be something more like `with tarfile.open(...) as tar:`? It looks like you could shave three lines off that way.